src/custom_base64.test.ts

```diff
@@ -0,0 +1,8 @@
+import { ValeriaEncode, ValeriaDecodeToPdchu } from "./custom_base64";
```

Comment:
ValeriaEncode isn't used.  Remove import or add a test that uses it as I believe this will cause a build error.

Reply:
Didnt saw a build error, only warning in ESLint. But its removed now